zproject/loaders.py

```diff
@@ -0,0 +1,17 @@
+from pathlib import Path
+from typing import List, Union
+
+from django.template.loaders import app_directories
+
+
+class TwoFactorLoader(app_directories.Loader):
+    def get_dirs(self) -> List[Union[str, Path]]:
+        dirs = super().get_dirs()
+        # app_directories.Loader returns only a list of
+        # Path objects by calling get_app_template_dirs
+        two_factor_dirs: List[Union[str, Path]] = []
+        for d in dirs:
+            assert isinstance(d, Path)
+            if d.match("two_factor/*"):
+                two_factor_dirs.append(d)
+        return two_factor_dirs
```

Comment:
Can we call this file `template_loaders.py`? The other option would be to put this in `zerver/lib/templates.py`; but I think the extra clarity is helpful for me at least in knowing what kind of thing is being loaded :).

Reply:
Sure.